diagnostics/install.md

```diff
@@ -0,0 +1,26 @@
+# Installer / Setup Logs
+Installer logs include information about any errors that WV2's installer/updater hit when trying to install or update the WV2 runtime. There are a few different logs that we need to best look in to any issues.
+
+## Registry Key
+This registry contains important update information like when the last update occurred, recent error codes, and updater version information.
+1. Open `regedit.exe`
```

Comment:
This instruction requires more work from end user. https://microsoft.visualstudio.com/Edge/_wiki/wikis/Edge.wiki/49240/WebView2-TSG?anchor=collect-edge-webview2-runtime-installation-info has command lines the end user can copy and paste to get the registry info easier. It also has instruction for collecting list of Edge files, which is sometimes useful (not always useful).

Reply:
Thanks, forgot about that page. Updated.